oauth2/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2020-Present Okta, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+logging.level.org.springframework.security.web=INFO
+logging.level.org.springframework.web.client.RestTemplate=DEBUG
+logging.level.com.okta.spring.boot.oauth=DEBUG
```

Comment:
Remove this file

Reply:
will do, kept it in place temporarily for debugging purposes.

Reply:
removed.